shell/platform/android/BUILD.gn

```diff
@@ -554,3 +558,63 @@ zip_bundle("android_javadoc") {
     ":gen_android_javadoc",
   ]
 }
+
+# Creates a local Maven repo structure under $root_out_dir/maven_repo
+maven_repo_dir = "maven_repo"
+maven_prefix = "$root_out_dir/$maven_repo_dir/io/flutter/"
+
+copy("flutter_embedding_repo_jar") {
+  sources = [ embedding_jar_path ]
+
+  outputs = [
+    "$maven_prefix/flutter_embedding_$flutter_runtime_mode/$_maven_version_string/{{source_name_part}}-$_maven_version_string.jar",
```

Comment:
I tried some different ways to make this work without explicitly spelling out the extensions and couldn't quite get it to work - `source_name_part` is a special thing for copy AFAICT in outputs, and I can't seem to use string_replace on it or get_path_info on it.